rosidl_typesupport_opensplice_c/cmake/rosidl_typesupport_opensplice_c_generate_interfaces.cmake

```diff
@@ -148,12 +148,6 @@ target_include_directories(${rosidl_generate_interfaces_TARGET}${_target_suffix}
   ${CMAKE_CURRENT_BINARY_DIR}/rosidl_generator_cpp
   ${CMAKE_CURRENT_BINARY_DIR}/rosidl_typesupport_opensplice_c
   ${CMAKE_CURRENT_BINARY_DIR}/rosidl_typesupport_opensplice_cpp
-  "${_output_path}/msg/dds_opensplice_c"
-  "${_dds_output_path}/msg/dds_opensplice"
-  "${_output_path}/srv/dds_opensplice_c"
-  "${_dds_output_path}/srv/dds_opensplice_c"
-  "${_output_path}/action/dds_opensplice_c"
-  "${_dds_output_path}/action/dds_opensplice_c"
```

Comment:
@dirk-thomas unrelated changes or am I missing something?

Reply:
Not unrelated at all. With the new option being used the generated files use full include paths and therefore these include directories are not needed anymore.